pbs/pbsresponse.go

```diff
@@ -1,26 +1,38 @@
 package pbs
 
+// PBSBid is a bid from the auction. These are produced by Adapters, and target a particular Ad Unit.
```

Comment:
Might be worth calling out that these produce bids in the format expected by prebid.js and prebid mobile SDK, so any format changes need to tie out to them